### PR TITLE
Add controller for PATCH request

### PR DESCRIPTION
**Pull request by Swastik19Nit**
solves  issue #14

### CODE REVIEW

File: server/src/controllers/blog-controller.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Blog from '../model/blog.js';
+
+export const updateBlog = async(req,res) =>{
+    try
+    {
+        const id=req.params.id;
```
Comment (Himasnhu-AT) [inline server/src/controllers/blog-controller.js]: try to concise this

File: server/src/controllers/blog-controller.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import Blog from '../model/blog.js';
+
+export const updateBlog = async(req,res) =>{
+    try
+    {
```
Comment (Himasnhu-AT) [inline server/src/controllers/blog-controller.js]: use this type of format: try {

ensure `{` is in same line, reducing the number of lines

File: server/src/controllers/blog-controller.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        {
+            return res.status(404).json({message:"Blog not found!"})
+        }
+
+        return res.status(200).json({message:'Blog Updated Successfully!'})
+
+    }
+    catch(error)
+    {
+        return res.status(500).json({message:"Error while updating Blog",error: error.message})
```
Comment (Himasnhu-AT) [inline server/src/controllers/blog-controller.js]: dont send error to fronend api as it is. Do proper sanitization or just console.error the message, and send `INTERNAL SERVER FAILURE` or something like that

**Review by Himasnhu-AT — CHANGES_REQUESTED** [review]
resolve these conversations @Swastik19Nit @raghzzzspace

**Comment by Swastik19Nit** [comment]
@Himasnhu-AT

**Review by Himasnhu-AT — APPROVED** [review]

**Pull request merged**